keps/sig-auth/3331-structured-authentication-configuration/README.md

```diff
@@ -724,49 +662,40 @@ providers such as Okta, Azure AD, etc:
 
 #### Beta
 
-- Gather feedback
 - Complete benchmarks
-- Add metrics
-- Initial e2e test with an external provider completed and enabled
+- Support > 1 JWT authenticator and > 1 audiences
+- Enable automatic reload of the configuration
 
 #### GA
 
+- Gather feedback
 - Add a full documentation with examples for the most popular providers, e.g., Okta, Dex, Auth0
 - Migration guide
 - Deprecation warnings for non-structured OIDC provider configuration
```

Comment:
Is this something that we actually want?

Reply:
I don't think we want this, so I've removed it.

I reviewed our initial [PR]( https://github.com/kubernetes/enhancements/pull/3332) and didn't see any discussion indicating a desire to do this either.